airdrop/application/services/user_claim_service.py

```diff
@@ -0,0 +1,70 @@
+from http import HTTPStatus
+
+import requests
+
+from airdrop.config import TokenTransferCardanoService, SLACK_HOOK
+from airdrop.constants import AirdropClaimStatus
+from airdrop.infrastructure.repositories.airdrop_repository import AirdropRepository
+from airdrop.infrastructure.repositories.claim_history_repo import ClaimHistoryRepository
+from common.logger import get_logger
+from common.utils import Utils
+
+logger = get_logger(__name__)
+utils = Utils()
+
+
+class UserClaimService:
+    def __init__(self, airdrop_id):
+        self.airdrop_id = airdrop_id
+        self.airdrop = AirdropRepository().get_airdrop_details(airdrop_id)
+        self.airdrop_class = self.airdrop.airdrop_processor
+
+    def prepare_token_transfer_cardano_service_payload(self, claims):
+        token = self.airdrop.token_name
+        from_address_wallet_name = self.airdrop_class(self.airdrop_id).claim_address
+        to_addresses = []
+        for claim in claims:
+            to_addresses.append({"address": claim.address, "amount": claim.claimable_amount})
```

Comment:
Replace with list comprehensions `[ {"address": claim.address, "amount": claim.claimable_amount} for claim in claims]`